packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -108,6 +97,17 @@ export const horizontalGalleryStyle = (shouldFloatLocalVideo: boolean, isNarrow:
   };
 };
 
+/**
+ * @private
+ */
+export const horizontalGalleryStyle = (isNarrow: boolean): HorizontalGalleryStyles => {
```

Comment:
Should this be in `HorizontalGallery.styles.tsx` instead?

Reply:
this is specific for ResponsiveHorizontalGallery inside VideoGallery, so this is placed correctly IMO (same as horizontalGalleryContainerStyle)

Reply:
I think it should stay in VideoGallery because this is the styles for the HorizontalGallery specific to VideoGallery